src/main/kotlin/net/atos/zac/app/documentcreation/DocumentCreationRestService.kt

```diff
@@ -9,98 +9,95 @@ import jakarta.inject.Inject
-    private val ztcClientService: ZtcClientService,
     private val zrcClientService: ZrcClientService,
-    private val enkelvoudigInformatieObjectUpdateService: EnkelvoudigInformatieObjectUpdateService,
-    private val restUnattendedDataConverter: RestDocumentCreationConverter
 ) {
     @POST
-    @Path("/createdocumentattended")
-    @Deprecated("Use createDocumentUnattended instead")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/create-document-attended")
     fun createDocumentAttended(
         @Valid restDocumentCreationAttendedData: RestDocumentCreationAttendedData
-    ): RestDocumentCreationAttendedResponse {
-        val zaak = zrcClientService.readZaak(restDocumentCreationAttendedData.zaakUUID).also {
+    ): RestDocumentCreationAttendedResponse =
+        zrcClientService.readZaak(restDocumentCreationAttendedData.zaakUuid).also {
             assertPolicy(policyService.readZaakRechten(it).creeerenDocument)
+        }.let {
+            DocumentCreationDataAttended(
+                zaak = it,
+                templateId = restDocumentCreationAttendedData.smartDocumentsTemplateId,
+                templateGroupId = restDocumentCreationAttendedData.smartDocumentsTemplateGroupId,
+                taskId = restDocumentCreationAttendedData.taskId,
+            )
+                .let(documentCreationService::createDocumentAttended)
+                .let { response -> RestDocumentCreationAttendedResponse(response.redirectUrl, response.message) }
         }
-        // documents created by SmartDocuments are always of the type 'bijlage'
-        // the zaaktype of the specified zaak needs to be configured to be able to use this informatieObjectType
-        return ztcClientService.readInformatieobjecttypen(zaak.zaaktype)
-            .firstOrNull { ConfiguratieService.INFORMATIEOBJECTTYPE_OMSCHRIJVING_BIJLAGE == it.omschrijving }?.let {
-                    informatieObjectType ->
-                DocumentCreationDataAttended(
-                    zaak = zaak,
-                    taskId = restDocumentCreationAttendedData.taskId,
-                    informatieobjecttype = informatieObjectType
-                ).let(documentCreationService::createDocumentAttended)
-                    .let { RestDocumentCreationAttendedResponse(it.redirectUrl, it.message) }
-            } ?: run {
-            throw InputValidationFailedException(
-                "No informatieobjecttype '${ConfiguratieService.INFORMATIEOBJECTTYPE_OMSCHRIJVING_BIJLAGE}' found for " +
-                    "zaaktype '${zaak.zaaktype}'. Cannot create document."
+
+    @POST
+    @Path("/redirect/zaak/{zaakUuid}")
+    fun createDocumentRedirect(
+        @PathParam("zaakUuid") zaakUuid: UUID,
+        @QueryParam("templateGroupId") templateGroupId: String,
+        @QueryParam("templateId") templateId: String,
+        @QueryParam("userName") userName: String,
+        @FormParam("sdDocument") smartDocumentId: String,
```

Comment:
Same here. I find the terms `sdDocument` and `smartDocumentId` a bit confusing..